apps/melding-form/src/app/(map)/locatie/kies/SelectLocation.test.tsx

```diff
@@ -14,7 +14,7 @@ vi.mock('./_components/AssetList/AssetList', () => ({
 
 vi.mock('@meldingen/map', () => ({
   Controls: vi.fn(),
-  Map: vi.fn(),
+  Map: vi.fn((props) => <div data-testid="map-mock">{props.children}</div>),
```

Comment:
I don't think this change is necessary for this PR?

```suggestion
  Map: vi.fn(),
```